tsconfig.json

```diff
@@ -4,7 +4,7 @@
         "inlineSourceMap": true,
         "inlineSources": true,
         "module": "ESNext",
-        "target": "es2018",
+        "target": "ES6",
```

Comment:
I've downgraded that to follow the [sample plugin](https://github.com/obsidianmd/obsidian-sample-plugin/blob/master/tsconfig.json)